code/modules/overmap/overmap_object.dm

```diff
@@ -2,13 +2,28 @@
 	name = "map object"
 	icon = 'icons/obj/overmap.dmi'
 	icon_state = "object"
+	color = "#c0c0c0"
 
 	var/known = 1				 //shows up on nav computers automatically
 	var/scannable				 //if set to TRUE will show up on ship sensors for detailed scans, and will ping when detected by scanners.
 
 	var/requires_contact = FALSE //whether or not the effect must be identified by ship sensors before being seen.
 	var/instant_contact  = FALSE //do we instantly identify ourselves to any ship in sensors range?
+	var/halted = FALSE
+	var/can_move = FALSE
 	var/sensor_visibility = 10	 //how likely it is to increase identification process each scan.
+	var/list/consoles
```

Comment:
why are consoles and map_z list needed on base level? Kinda against the point of visitable sectors being a subtype.